conf/app.routes

```diff
@@ -369,14 +369,14 @@ GET         /:srn/submit-file-upload-too-many-rows
 GET         /:srn/change-file-upload-too-many-rows                                      controllers.nonsipp.memberdetails.upload.FileUploadTooManyRowsController.onPageLoad(srn: Srn, mode: Mode = CheckMode)
 GET         /:srn/change-submit-file-upload-too-many-rows                               controllers.nonsipp.memberdetails.upload.FileUploadTooManyRowsController.onSubmit(srn: Srn, mode: Mode = CheckMode)
 
-GET         /:srn/partnership-recipient-utr                                             controllers.nonsipp.common.PartnershipRecipientUtrController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
-GET         /:srn/partnership-recipient-utr/:index                                      controllers.nonsipp.common.PartnershipRecipientUtrController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
-POST        /:srn/partnership-recipient-utr                                             controllers.nonsipp.common.PartnershipRecipientUtrController.onSubmit(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
-POST        /:srn/partnership-recipient-utr/:index                                      controllers.nonsipp.common.PartnershipRecipientUtrController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
-GET         /:srn/change-partnership-recipient-utr                                      controllers.nonsipp.common.PartnershipRecipientUtrController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = CheckMode)
-GET         /:srn/change-partnership-recipient-utr/:index                               controllers.nonsipp.common.PartnershipRecipientUtrController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
-POST        /:srn/change-partnership-recipient-utr                                      controllers.nonsipp.common.PartnershipRecipientUtrController.onSubmit(srn: Srn, index: Max5000 = 1, mode: Mode = CheckMode)
-POST        /:srn/change-partnership-recipient-utr/:index                               controllers.nonsipp.common.PartnershipRecipientUtrController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
+GET         /:srn/partnership-recipient-utr-:subject                                             controllers.nonsipp.common.PartnershipRecipientUtrController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode, subject: IdentitySubject)
```

Comment:
Subject can be long. And it is not necessary a recipient. So I'd shorten the first part. E.g. `/utr-of-:subject`